### PR TITLE
15 add timer UI replacement the the built in m+ timer

### PR DESCRIPTION
**Pull request by Larsj02**

### CODE REVIEW

File: constants/misc.lua
Comment (kaelonR) [inline constants/misc.lua:1]: This file was moved to constants/styles.lua. Please remove this file from your branch.

File: assets/textures/rounded-square-border-10px.tga
Comment (kaelonR) [inline assets/textures/rounded-square-border-10px.tga:1]: This has been moved to assets/textures/rounded-frame/border-10px.tga. Please remove this file from your branch.

File: assets/textures/rounded-square-border-1px.tga
Comment (kaelonR) [inline assets/textures/rounded-square-border-1px.tga:1]: This has been moved to assets/textures/rounded-frame/border-1px.tga. Please remove this file from your branch.

File: assets/textures/rounded-square-border-2px.tga
Comment (kaelonR) [inline assets/textures/rounded-square-border-2px.tga:1]: This has been moved to assets/textures/rounded-frame/border-2px.tga. Please remove this file from your branch.

File: assets/textures/rounded-square-border-4px.tga
Comment (kaelonR) [inline assets/textures/rounded-square-border-4px.tga:1]: This has been moved to assets/textures/rounded-frame/border-4px.tga. Please remove this file from your branch.

File: assets/textures/rounded-square-border-6px.tga
Comment (kaelonR) [inline assets/textures/rounded-square-border-6px.tga:1]: This has been moved to assets/textures/rounded-frame/border-6px.tga. Please remove this file from your branch.

File: assets/textures/rounded-square-border-8px.tga
Comment (kaelonR) [inline assets/textures/rounded-square-border-8px.tga:1]: This has been moved to assets/textures/rounded-frame/border-8px.tga. Please remove this file from your branch.

File: assets/textures/rounded-square.tga
Comment (kaelonR) [inline assets/textures/rounded-square.tga:1]: This has been moved to assets/textures/rounded-frame/base.tga. Please remove this file from your branch.

**Review by Thekinghim — APPROVED** [review]

**Review by kaelonR — APPROVED** [review]

**Pull request merged**